jsonpickle/pickler.py

```diff
@@ -539,6 +541,7 @@ def _flatten_obj_instance(self, obj):
         has_getnewargs_ex = util.has_method(obj, '__getnewargs_ex__')
         has_getinitargs = util.has_method(obj, '__getinitargs__')
         has_reduce, has_reduce_ex = util.has_reduce(obj)
+        self._excludes = set(getattr(obj, '_jsonpickle_exclude', ()))
```

Comment:
Now that I'm re-reading this, it dawned on me that we're storing the excludes on the entire pickler instance but the excludes should probably really only affect the current object being pickled.

If y'all agree, I think we should eliminate the `self._excludes` attribute and the associated constructor argument.

What we should instead do is change this part to use a local variable `excludes = set(....)` that we can pass to `_flatten_dict_obj()`. We should change that function's signature to have an `exclude` keyword argument instead of reading it from `self._exclude`.

Another nice benefit of this is that we won't end up changing the constructor's signature.

Reply:
I think that's a good point. After all, why would we set an ``_exclude`` attribute if we're just going to overwrite it anyway (rhetorically)?

The only possible downside I can see is that for users who have to serialize lots of small objects (which is my use-case for jsonpickle in my hobby projects), having to set an extra attribute on the ``__slots__`` or ``__dict__`` for every instance could lead to a non-insignificant slowdown and especially extra memory usage, compared to setting it once on the Pickler instance. I'll fire up a benchmark real quick and edit in the results when done.